### PR TITLE
address rubocop issues for dropbox driver

### PR DESCRIPTION
**Pull request by derekmorr**
Address several rubocop issues in the dropbox driver. This removes ACB size, method size, and timezone-parsing issues.

### CODE REVIEW

File: lib/browse_everything/driver/dropbox.rb [hunk 1]
```diff
@@ -3,39 +3,45 @@
...
       def icon
         'dropbox'
       end
 
       def validate_config
-        unless [:app_key, :app_secret].all? { |key| config[key].present? }
-          raise BrowseEverything::InitializationError, 'Dropbox driver requires :app_key and :app_secret'
-        end
+        return if CONFIG_KEYS.all? { |key| config[key].present? }
+        raise BrowseEverything::InitializationError, 'Dropbox driver requires :app_key and :app_secret'
```
Comment (jeremyf) [inline lib/browse_everything/driver/dropbox.rb]: Leverage the new constant to build the exception message:

```ruby
raise BrowseEverything::InitializationError, "Dropbox driver requires #{CONFIG_KEYS.inspect}"
```

Comment (derekmorr) [inline lib/browse_everything/driver/dropbox.rb]: Adderessed in 5c819fa

**Comment by jeremyf** [comment]
Thank you for your work @derekmorr; I looked and do not see that you have an individual contributor license agreement (iCLA) on file.  For your code to be merged, we require a signed iCLA. 

Contributing Guidelines: https://github.com/projecthydra/browse-everything/blob/master/CONTRIBUTING.md#hydra-project-intellectual-property-licensing-and-ownership

**Comment by derekmorr** [comment]
I can submit an iCLA if needed. I'm not sure if it is -- I'm a Penn State employee. Although I'm not a part of the PSU Hydra team, their offices are next door to mine, and we often work on refactoring projects together. Adam Wead and I jointly worked on #153 , for example. So I may be covered by the PSU CLA. If not, I'll submit an iCLA. Let me know how you'd like me to proceed.

**Comment by jeremyf** [comment]
@derekmorr we will still need an iCLA. Project Hydra has Penn States institutional contributor license agreement on file.

Basically, your personal iCLA says "I agree to share my code". The institutional CLS says "We agree to share code contributed on behalf of our employees (so long as they personally agree as well)."

The iCLA is a CYA for you with your employer. You can talk to @DanCoughlin to submit an iCLA.

**Comment by mjgiarlo** [comment]
@derekmorr You can find the iCLA here: https://wiki.duraspace.org/download/attachments/31662457/Hydra%20Project%20Contributor%20License%20Agreement%20for%20Individuals-20121216.pdf?version=1&modificationDate=1355704453858&api=v2 If you sign it and send it along to legal@projecthydra.org, I can get it processed today. Thanks!

If you're curious about why Hydra requires both a corporate and an individual CLA, we cover that a bit here: https://github.com/escowles/technical-onboarding/blob/master/formalities.md#what-is-a-cla-and-why-do-i-need-one

Thanks for the contribution, Derek. :)

**Comment by derekmorr** [comment]
Ok Mike, will do. We're snowed in here, so it'll be at least tomorrow until I can get to the office to print it out and scan it.

**Comment by derekmorr** [comment]
iCLA submitted today. You should get notification soon.

**Comment by mjgiarlo** [comment]
@derekmorr OK, great. I'll process it soon and ping you back over email.

**Comment by jeremyf** [comment]
@mjgiarlo Does this mean I can approve and merge?

**Comment by mjgiarlo** [comment]
@jeremyf not yet, no

**Comment by mjgiarlo** [comment]
@jeremyf You're clear to approve and merge.

@derekmorr Welcome to the Hydra community, sir. :smile:

**Pull request merged**